components/klaudsolcms/AppContentBuilderTable.js

```diff
@@ -41,7 +42,7 @@ const AppContentBuilderTable = ({typeSlug}) => {
       (async () => {
         try {
           const valuesRaw = await slsFetch(`/api/entity_types/${typeSlug}/attributes/${attribute?.name}`, {
-            method: "DELETE"
+            method: "DELETE",
```

Comment:
What's the comma for?
```suggestion
            method: "DELETE"
```

Reply:
Oh, it's either my formatter, or when I removed the authorization headers, I accidentally removed the method and when I put it back, I entered a comma out of habit. Anyway, I removed the comma :)